Automattic/jetpack
# Search: show "Jetpack Search Free" on my plans

Fixes #26999

#### Changes proposed in this Pull Request:

This PR adds "Jetpack Search Free" to the list of products on Jetpack -> Dashboard -> My plan tab in full Jetpack plugin:

![image](https://user-images.githubusercontent.com/6437642/197275123-b4d73d9c-c7e4-4d00-b559-591b9d511948.png)


#### Other information:


#### Jetpack product discussion

p1666362454158339-slack-C02ME06LF

#### Does this pull request change what data or activity we track or use?

No



#### Testing instructions:






* Activate full Jetpack plugin
* Activate Jetpack Search Free plan
* Go to  Jetpack -> Dashboard -> My plan tab
* Verify that "Jetpack Search Free" is visible on the products list
* Remove Jetpack Search Free
* Do the same steps for regular Jetpack Search, verify that it's still listed as "Jetpack Search" (without free) on the products list
